.editorconfig

```diff
@@ -34,3 +34,7 @@ indent_size = 2
 [CMakeLists.txt]
 indent_style = space
 indent_size = 2
+
+[*.scss]
+indent_style = tab
+indent_size = 4
```

Comment:
`tab` is already in the default entry `*`, which should cover everything that isn't listed explicitly. Also, there shouldn't be a reason to set an `indent_size` for tabs - after all, the main advantage of tab indentation is that every developer change change the tab width according to their own preferences. So as far as I can tell, this commit can be dropped entirely.

Reply:
no problem. i will remove this completely.

Reply:
fixed with 0f89c6b13b1f856b72c3b37e7198659cb85a3a30